sage-ios/SAGE/SAGE/BrowseMentorsTableViewCell.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  BrowseMentorsTableViewCell.swift
+//  SAGE
+//
+//  Created by Sameera Vemulapalli on 11/9/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+//
+
+import UIKit
+
+class BrowseMentorsTableViewCell: UITableViewCell {
+    
+    var mentorPicture = UIImageView()
+    var mentorName = UILabel()
+    var schoolName = UILabel()
+    var totalHours = UILabel()
+    let user: User
```

Comment:
this is very very bad. never make a model a property of a tableviewcell because cells get reused. instead, this cell should have some method like configureWithUser(_:) which will configure the cell with the user